src/test/scenarios/boolean/output/src/boolean/README.md

```diff
@@ -1,5 +1,12 @@
-Microsoft Azure CLI 'boolean' Extension
-==========================================
+# Azure CLI boolean Extension #
+This is a extension for boolean
 
-This package is for the 'boolean' extension.
-i.e. 'az boolean'
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name boolean
+```
+
+### Included Features ###
+#### boolean bool: ####
```

Comment:
If no examples, should we remove ### Included Features ### part?

Reply:
Leave here as it's more like a kind of template. Template will list all command group here even there are no example. It can also be a hint that some example is needed here. Customer can add more sample if needed. If no sample, just remove it manually.